### DISCUSSION TITLE
面板“概览”页面的“监控”栏、“状态”栏以及右侧“推荐应用”栏为空白

### DISCUSSION THREAD

**Discussion by QingyeSC** [1Panel-dev/1Panel#6162] [Q&A]
### 联系方式

218268cs@gmail.com

### 1Panel 版本

社区版：v1.10.13-lts

### 问题描述

正常运行面板快40天后，面板的概览”页面的“监控”栏、“状态”栏以及右侧“推荐应用”栏突然显示为空白，当时在尝试安装 ClamAV 服务，但之后删除 ClamAV 服务相关文件后，就空白了
![屏幕截图 2024-08-16 230046](https://github.com/user-attachments/assets/24904c20-ea69-48a6-84f7-d6eabd6bb9dc)

### 重现步骤

无

**Comment by wanghe-fit2cloud** [comment]
提供一下系统日志截图。

**Reply by QingyeSC** [reply]
我换成Edge浏览器访问就一切正常了，使用chrome浏览器，无论怎么清除cookie或者重新登录都不行，奇怪
